content/guides/trash/index.md

```diff
@@ -0,0 +1,17 @@
+---
+rank: 50
```

Comment:
Please change to 150 for now. It's the arbitrary order we assigned for now according to the Master Categories tab of the spreadsheet